invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -591,6 +591,10 @@ code {
   display: contents !important;
 }
 
+.top-0 {
```

Comment:
I would rather prefer you add top:0 on `#invenio-menu` element
the reason is that top property cannot be used on its own properly (it comes with `position:` and if position is not set correctly the `top:` won't work, therefore adding it as a standalone class can be confusing for developers